examples/yew-tailwindcss/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>Trunk | Yew | Tailwind</title>
+    <link rel="stylesheet" href="tailwind.css"/>
+
+    <link data-trunk rel="scss" href="src/index.scss"/>
+    <link data-trunk rel="scss" data-inline href="src/inline-scss.scss"/>
+    <link data-trunk rel="css" href="src/app.css"/>
+    <link data-trunk rel="icon" href="src/yew.svg"/>
+    <link data-trunk rel="copy-file" href="src/yew.svg"/>
+    <base data-trunk-public-url/>
+</head>
+<body>
+    <link data-trunk rel="rust" href="Cargo.toml" data-bin="yew-example" data-cargo-features="demo-abc,demo-xyz"/>
```

Comment:
Same thing here. We should be able to just remove this line and allow the defaults. If you want to keep the line, then let's fix the `data-bin` name and remove the cargo features.

Reply:
Removed